src/graphql/resolvers.ts

```diff
@@ -19,6 +19,8 @@ export const hashPassword = async (password: string) => {
   return hashedPassword;
 };
 
+export const defaultSearchValue = 10;
```

Comment:
```suggestion
export const defaultLimit = 10;
```

O nome `defaultSearchValue` me passa impressão de um parâmetro de busca, não exatamente do número de usuários, concorda?